app/src/main/java/com/github/orelzion/spotifyclone/view/AlbumListFragment.kt

```diff
@@ -1,56 +1,72 @@
 package com.github.orelzion.spotifyclone.view
 
 import android.os.Bundle
-import android.util.Log
 import android.view.View
-import android.widget.Toast
+import android.widget.ProgressBar
+//import android.widget.Toast
+import androidx.core.view.isVisible
 import androidx.fragment.app.Fragment
-import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.commit
 import androidx.recyclerview.widget.RecyclerView
 import com.github.orelzion.spotifyclone.R
-import com.github.orelzion.spotifyclone.model.AlbumsResponse
-import com.github.orelzion.spotifyclone.model.repository.BrowseRepository
-import com.github.orelzion.spotifyclone.viewmodel.AlbumViewData
+//import com.github.orelzion.spotifyclone.model.AlbumsResponse
+//import com.github.orelzion.spotifyclone.model.repository.BrowseRepository
```

Comment:
Please remove unused import, don't keep them in comments, it's a good practice to not leave commented code when submitting a PR

Reply:
Oh
That's so obvious. I meant to remove them after I make sure the app runs fluently, and remove it prior submitting; I missed it..